apps/generic_terminal/unix_utilities.talon

```diff
@@ -0,0 +1,4 @@
+tag: terminal
```

Comment:
I wouldn't include the 'terminal' tag here, just your custom unix_utilities one. That gives users the flexibility to add it in to e.g. their IDE which might have an embedded terminal but not otherwise be tagged as one.

Reply:
Done